<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>car_edit</class>
 <widget class="QDialog" name="car_edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>렌터카 등록</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="edit_btn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>수정</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>301</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>모델</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>일일요금</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>차종</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>수량</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>연료</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="model_txt"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="quantity_txt"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="charge_txt"/>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="fuel_combo">
      <item>
       <property name="text">
        <string>휘발유</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>경유</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LPG</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>전기</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="type_combo">
      <item>
       <property name="text">
        <string>소형</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>중형</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>대형</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SUV</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>등록번호</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="num_txt">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="add_btn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>추가</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
